src/main/java/ua/softserveinc/tc/validator/TimeValidator.java

```diff
@@ -32,11 +32,17 @@ public void validate(Object target, Errors errors) {
         }
     }
 
-    public boolean validateDate(String date)
-    {
+    public boolean validateDateFormat(String date) {
         return date.matches(ValidationConstants.DATE_REGEX);
     }
 
+    public boolean validateDate(String startTime, String endTime) {
+        if(validateDateFormat(startTime) && validateDateFormat(endTime)) {
+            return startTime.compareTo(endTime) <= 0;
```

Comment:
emm.. got stuck with that..
if startTime.compareTo(endTime) returns 0, that whole method returns true.. 

is it should be possible to enter startDate which is equals to endDate?